lib/plugins/aws/deploy/lib/uploadArtifacts.js

```diff
@@ -6,24 +6,33 @@ const filesize = require('filesize');
 const path = require('path');
 
 module.exports = {
-  uploadCloudFormationFile() {
-    this.serverless.cli.log('Uploading CloudFormation file to S3...');
+  uploadCloudFormationFiles() {
+    this.serverless.cli.log('Uploading CloudFormation files to S3...');
 
-    const compiledTemplateFileName = 'compiled-cloudformation-template.json';
+    const serverlessDirPath = path.join(this.serverless.config.servicePath, '.serverless');
+    const cfFileNameRegex = this.provider.naming.getCloudFormationFileNameRegex();
```

Comment:
The SLS owned CF files should not be grabbed by regex here. Other plugins might add CF files to the .serverless directory, that do not belong to the SLS upload, but will be uploaded independently by the plugin.